bindings/azure/blobstorage/blobstorage.go

```diff
@@ -360,50 +412,52 @@ func (a *AzureBlobStorage) list(ctx context.Context, req *bindings.InvokeRequest
 		}
 		hasPayload = true
 	}
-
 	if hasPayload {
-		options.Details.Copy = payload.Include.Copy
-		options.Details.Metadata = payload.Include.Metadata
-		options.Details.Snapshots = payload.Include.Snapshots
-		options.Details.UncommittedBlobs = payload.Include.UncommittedBlobs
-		options.Details.Deleted = payload.Include.Deleted
+		options.Include.Copy = payload.Include.Copy
+		options.Include.Metadata = payload.Include.Metadata
+		options.Include.Snapshots = payload.Include.Snapshots
+		options.Include.UncommittedBlobs = payload.Include.UncommittedBlobs
+		options.Include.Deleted = payload.Include.Deleted
 	}
 
 	if hasPayload && payload.MaxResults != int32(0) {
```

Comment:
Should probably check to make sure that `payload.MaxResults` is > 0?

Nit: I don't think the int32() casting is needed